packages/experiments/src/components/PersonaCoin/PersonaCoinPage.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { ExampleCard, IComponentDemoPageProps, ComponentPage, PropertiesTableSet } from '@uifabric/example-app-base';
+
+import { PersonaCoinExample } from './examples/PersonaCoin.Example';
+import { PersonaCoinSizeAndColorExample } from './examples/PersonaCoinSizeAndColor.Example';
+
+const PersonaCoinExampleCode = require('!raw-loader!@uifabric/experiments/src/components/PersonaCoin/examples/PersonaCoin.Example.tsx') as string;
+const PersonaCoinSizeAndColorExampleCode = require('!raw-loader!@uifabric/experiments/src/components/PersonaCoin/examples/PersonaCoinSizeAndColor.Example.tsx') as string;
+
+export class PersonaCoinPage extends React.Component<IComponentDemoPageProps, {}> {
```

Comment:
>PersonaCoinPage [](start = 13, length = 15)

nit: can be a functional component

Reply:
Yes indeed, copy pasted from another component :P

Reply:
True. Copy pasted this from another component :)